packages/runtime-stack/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@omakase/runtime-stack",
+  "version": "0.0.1",
+  "description": "A collection of dependencies commonly used across Artsy projects",
+  "repository": "https://github.com/omakase-js/omakase/tree/master/packages/dev-stack",
+  "author": "Artsy Inc.",
+  "license": "MIT",
+  "dependencies": {
+    "@artsy/palette": "^4.14.7",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-relay": "^4.0.0",
+    "styled-components": "^4.3.1",
+    "styled-system": "^5.0.5"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.4.5",
+    "prop-types": "^15.7.2",
+    "react-native": "^0.55.4",
```

Comment:
I think we should probably remove `react-native` from this list. Maybe web/native could have different runtime stacks?

The reason why I say that is that ts sometimes has a really bad tendency to choke on types from react-native when react-dom is also present. You have to do weird things like have a post install script that remove's react-native's type definition file.

Reply:
This was only added to satisfy a peerDependency warning from Palette, so its of no harm to remove.